guides/keras_hub/semantic_segmentation_deeplab_v3.py

```diff
@@ -0,0 +1,845 @@
+"""
+Title: Semantic Segmentation with KerasHub
+Author: [Sachin Prasad](https://github.com/sachinprasad)
```

Comment:
If this guide was adapted from KerasCV guide please also mention the original authors here.